scripts/disableAz.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Usage: disable and renable an AZ
+#
+# prereq
+#  - jq
+#  - aws-cli
+
+# AZ to disable
+AZ=eu-west-1a
```

Comment:
I was trying it out with a different region than I have stored in `~/.aws/config`

i.e. content of my `~/.aws/config`:
```
[default]
region = eu-central-1
```

and the script didn't work, because all aws commands are automatically targeting `eu-central-1` region

I think if all aws commands in this script would contain additional parameter `--region ${AZ:0:-1}`, it should work regardless what user has stored in `~/.aws/config`. Wdyt?

Also, do you think it would make sense to add the AZ as an optional parameter? (i.e. to use it like this: `./scripts/disableAz.sh --az eu-central-1c`)

Reply:
Thanks @psturc , yea makes sense.